### PR TITLE
Issue #14: NavBar Component

### LINKED ISSUE #14: NavBar Component
Create navbar component for the front page.

mentor - @OmGuptaIND

### PR DESCRIPTION
**Pull request by arcinston**
Made a basic navbar with the new logo made by @Ayush-Shukla02  and did some css styling

### CODE REVIEW

**Comment by OmGuptaIND** [comment]
Use the colours from: root selector in index.css don't use other colours

and change code to 

`nav{
  display: flex;
  justify-content: space-between;
  padding: 10px 30px;
}
.logo{
  display: flex;
  align-items: center;
}
.logo>img{
  margin-right: 10px;
  width: 50px;
  border-radius: 50%;
}
.nav-options{
  display: flex;
  align-items: center;
}
.nav-options>a{
  padding: 10px;
  margin: 5px;
}`

change the code to these please or see the code here and make the changes

**Comment by OmGuptaIND** [comment]
HTML

nav{
  display: flex;
  justify-content: space-between;
  padding: 10px 30px;
}
.logo{
  display: flex;
  align-items: center;
}
.logo>img{
  margin-right: 10px;
  width: 50px;
  border-radius: 50%;
}
.nav-options{
  display: flex;
  align-items: center;
}
.nav-options>a{
  padding: 10px;
  margin: 5px;
}

@arcinston

**Comment by OmGuptaIND** [comment]
and also this merge has conflicts see why they are happening and how can they be resolved

**Comment by AkMo3** [comment]
GitHub, generate site

**Comment by AkMo3** [comment]
I guess aarush needs to comment it for it to work. The page generated consist of already generated github page.

**Comment by arcinston** [comment]
@OmGuptaIND look at the code now , i made the css cleaner but i need .NavMenu and .ProjectName for easy changes in css

File: package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "homepage": "https://arcinston.github.io/stonks-frontend",
```
Comment (AkMo3) [inline package.json]: Ye line hata.

**Comment by AkMo3** [comment]
Abhi ek kam kar bas, sare changes squash merge kar. `package-lock.json` mat track kar. `npm install` karne par vo apne aap aajaega local par, so no need to update it. Waise mai .gitignore file banakar daal deta hu, taki aage apne aae ye changes.

**Comment by OmGuptaIND** [comment]
Work done

**Pull request closed without merging**